index.html

```diff
@@ -62,13 +73,14 @@ <h2>Moon</h2>
         </section>
         
         <footer>
-            <nav>
-                <a href="#">Home</a>
-                <a href="#">About</a>
-                <a href="#">Products</a>
-                <a href="#">Blog</a>
-                <a href="#">Contact</a>
-            </nav>
+                <nav>
+                    <a href="#">Home</a>
+                    <a href="./about.html">About</a>
```

Comment:
Excellent. Also try to add the links on the nav in about page
Alternatively, try to see how you can reuse the nav and header by leaving them in a single page